### ISSUE TITLE
Make the first heading in App pages h1

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeCENMigration#1389]
May be disconcerting for screen reader users that the refinery pages start with h1 tags, app pages start with h2. tags.

Pat to investigate the impact of the current situation for users of screen readers (check webaim archives to start with).

Alison to see how webaim pages themselves are marked up.

**Comment by AlOneill** [comment]
WebAIM mark up their pages in an unexpected way: the heading with the logo in the header is an `h2` element whilst the first heading in the page main content is an `h1`. Cannot see an explanation of this markup practice in their information pages.

I will investigate further …

**Comment by AlOneill** [comment]
It looks like most sites use an `h1` element to identify the main content of an individual page.

However, whether or not there is a heading element (of any level) in the header material (brand logo, main navigation and so forth) is highly variable. Of course, if the logo is a link, then it **must not be empty**, which the CEN one is — that is an error.

**Comment by DeniseColbert** [comment]
All pages should have an h1 at the top of the main content. E.g.

![image](https://user-images.githubusercontent.com/19173257/162207778-91f9b3cc-61da-4490-a687-3d10e4f35206.png)


Search results - Hidden h1 
Detailed results - Hidden h1

All drilled-down pages in Records section should have an h1 specific to that page.

**Comment by PatReynolds** [comment]
Records pages being reviewed by @AlOneill @AnneV-Learn 

@AnneV-Learn to do other pages after the review.
